src/Modules/Timing.asm

```diff
@@ -691,8 +683,6 @@ comp_read_wrong_startup_jump:
     sjmp comp_check_timeout             ; Continue to look for good ones
 
 comp_read_wrong_extend_timeout:
-    clr  Flag_Demag_Detected            ; Clear demag detected flag
```

Comment:
Why is it no longer necessary to clear the demag detection flag?

Reply:
Good catch

Reply:
Fixed

Reply:
Fixed